WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -147,12 +148,17 @@ class PageListViewModel @Inject constructor(
         pagesViewModel.onNewPageButtonTapped()
     }
 
-    fun onScrollToPageRequested(remotePageId: Long) {
-        val position = _pages.value?.indexOfFirst { it is Page && it.remoteId == remotePageId } ?: -1
+    fun setPageToScrollTo(pageId: LocalId) {
+        pageToScrollTo = pageId
+    }
+
+    private fun onScrollToPageRequested(localPageId: Int) {
+        val position = _pages.value?.indexOfFirst { it is Page && it.localId == localPageId } ?: -1
         if (position != -1) {
             _scrollToPosition.postValue(position)
+            pageToScrollTo = null
```

Comment:
I am clearing the `pageToScrollTo` here only when the id was found due to times when the scroll wouldn't take place. I am not sure about this and it's mutating state in a different location so I need your thoughts on this and I will double-check it. Didn't want it to stop the review though.